code/modules/antagonists/zombie/zombie_spells.dm

```diff
@@ -29,14 +35,14 @@
 /datum/spell/zombie_claws/proc/dispel()
 	var/mob/living/carbon/human/user = action.owner
 	var/obj/item/zombie_claw/claw = user.get_active_hand()
-	if(istype(claw, /obj/item/zombie_claw))
+	if(istype(claw, /obj/item/zombie_claw || /obj/item/zombie_claw/plague_claw))
```

Comment:
I'm not sure this works how you'd expect it to, since the result of the boolean comp will result in `istype(claw, TRUE)`
```suggestion
	if(istype(claw, /obj/item/zombie_claw) || istype(claw, /obj/item/zombie_claw/plague_claw))
```